cms/djangoapps/contentstore/utils.py

```diff
@@ -2149,11 +2149,51 @@ def get_validation_messages(xblock):
         xblock: The xblock object to validate.
 
     Returns:
-        tuple:
-            - validation_errors (list): A list of validation error messages.
-            - has_validation_error (bool): True if there are validation errors, False otherwise.
+        list: A list of validation error messages.
     """
     validation_json = xblock.validate().to_json()
-    validation_errors = validation_json['messages']
-    has_validation_error = bool(validation_errors)
-    return validation_errors, has_validation_error
+    return validation_json['messages']
+
+
+def get_render_error(request, xblock):
```

Comment:
Maybe we can call it `get_xblock_render_error` and the function that is inside will be called `get_xblock_render_error_context`.

This will allow to put the inner function outside and clarify the idea behind these two functions.

Reply:
I don't think that we need to move out this function, it will be used only here.
Renamed